### ISSUE TITLE
Vendor prefixes & duplicate properties with improper(?) constant usage

### ISSUE THREAD

**Issue by ndreckshage** [zeit/styled-jsx#231]
I use vars like this to bring in multiple properties easily...

```
export const latoRegular = 'font-family: 'Lato', sans-serif; font-weight: 400;'
```

```
.x {
  ${latoRegular} 
  font-size: 2rem;
}
```

I'm not sure if this was intended (I don't see tests doing it), but it works. However, I am now getting duplicate properties and vendor prefixes on properties that shouldn't have them. See screenshot.

<img width="1091" alt="screen shot 2017-06-08 at 1 04 20 am" src="https://user-images.githubusercontent.com/970587/26914399-915cb3e8-4be6-11e7-9d8c-e0539179e38e.png">

Noticed this upgrading from *0.5.7* to *1.0.4*. Was not an issue previously.

**Comment by giuseppeg** [comment]
thanks @ndreckshage for reporting this, I am working on a fix #227 which should go out today or tomorrow

**Comment by ndreckshage** [comment]
thank you! great project btw, really enjoy using styled-jsx.

**Comment by thysultan** [comment]
@giuseppeg Do you know why it happens.

**Comment by giuseppeg** [comment]
In 1.0.4 expressions are replaced with a custom property `var(--styled-jsx-placeholder-1)`, but that doesn't work in every use cases which is what I am fixing in #227. Probably stylis is confused also about floating custom props like

```css
.blue {
  font-size: 2rem;
  var(--styled-jsx-placeholder-1)
}
```

In general it is probably not a good idea to use constants for `property: value` pairs. Better do:

```js
const blue = '#00f'

`
.blue {
  font-size: 2rem;
  color: ${blue}
}
`
```

I'll see if #227 fixes this issue

**Comment by giuseppeg** [comment]
should be fixed in #227, happy to reopen if that is still an issue (wait for the new release)
